app/views/donations/new.html.erb

```diff
@@ -1,4 +1,10 @@
-<%= form_tag [:donations] do %>
+<%= form_tag [:donations], id: "js-stripe-form" do %>
```

Comment:
I don't know for form that are for external library or other thing like that, but we try to use `form_with` for the form. If it's specific to Stripe or there is something that you need to do that can be do with `form_with` it's cool. Otherwise, consistency would be nice. :)

Reply:
Totally cool! I've been in Rails 4.2 land for a while now with work and just wasn't thinking about it. I'm sure that `form_with` should be just fine. 👍